### DISCUSSION TITLE
Welcome to our 2022 GSoC Student!

### DISCUSSION THREAD

**Discussion by rcoup** [koordinates/kart#628] [Google Soc / SoD]
We are proud to announce that @amal-thundiyil has been selected to work on a Kart project for GSoC 2022. 🎉

Amal will be working on improving the experience of the CLI users of Kart by creating a framework for generating, distributing, and viewing documentation with Kart and also providing tab completion for Kart commands, options, branches, tags, files, metadata, features, and other context-relevant information — across all the platforms we support.

Thank you to everyone who got in touch, asked questions, and submitted proposals for GSoC with Kart this year — as a new organisation to GSoC we didn't really know what to expect and we appreciate your efforts and attention.

Amal will be posting progress updates over the next few months as he gets underway

Rob 😃

**Comment by amalthundiyil** [comment]
Hello, everyone! I am delighted to be working on [Kart](https://github.com/koordinates/kart) under the mentorship of [Robert Coup](https://github.com/rcoup), [Craig de Stigter](https://github.com/craigds) and [Andrew Olsen](https://github.com/olsen232) for the project - [Kart CLI Help Improvements](https://summerofcode.withgoogle.com/programs/2022/projects/EfCc3ywr) as part of the Google Summer of Code. I will be sharing my experience [here](https://amalthundiyil.hashnode.dev/google-summer-of-code-22-with-kart-project).

**Comment by Binay-tech1608** [comment]
Hi, I'm Binay Mann. I am a currently a 3rd year ug student pursuing btech in CSE. I want to join your organization and contribute to your projects. So please guide me in my open source journey.
